internal/services/applications/application_data_source.go

```diff
@@ -42,6 +43,12 @@ func applicationDataSource() *schema.Resource {
 				ValidateDiagFunc: validate.UUID,
 			},
 
+			"disabled_by_microsoft_status": {
```

Comment:
i don't think we need status?
```suggestion
			"disabled_by_microsoft": {
```